modules/nw-dns-cache-tuning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-cache-tuning_{context}"]
+= Tuning the CoreDNS cache
+
+You can configure the maximum duration of both successful (positive) or unsuccessful (negative) caching done by CoreDNS. Tuning the duration of caching of DNS query responses can reduce the load for any upstream DNS resolvers.
+
+.Procedure
+
+. Edit the DNS Operator object named `default` by running the following command:
++
+[source,terminal]
+----
+$ oc edit dns.operator.openshift.io/default
+----
+
+. Modify the time-to-live (TTL) caching values:
++
+.Configuring DNS caching
+[source,yaml]
+----
+spec:
+  cache:
+    successTTL: 1h <1>
+    denialTTL: 0.5h10m <2>
+----
++
+<1> The string value `1h` is converted to its respective number of seconds by CoreDNS. If this field is left empty, the value is assumed to be `0s` and the cluster will use the internal default value of `900s` as a fallback.
```

Comment:
Does this also support `1m`, `1d`, ect? If so should we mention that the specific string is some combination of days, hours, minutes, and seconds? I assume this is using the standard Go lang tokens for these?

Reply:
It is using the standard Go time units with the max time unit being `h` and it supports mixed units so you could do `1h30m10s` or `0.5h`.

Reply:
Technically, you can use the smaller time units like `ms` but we don't want to encourage users to do that since the functional lowest value is `1s` and that's unlikely to be used except in maybe a troubleshooting scenario.

Reply:
Thanks Chad! I think since it uses standard Go time units, if we expect users to be familiar with this, I will leave it as is. The other CoreDNS docs don't mention anything about Go, they just link to the relevant CoreDNS doc.